core/src/i18n/en.yaml

```
@@ -964,6 +965,16 @@ instances:
   description: <0>Service Instance</0> is a provisioned instance of a service that can be consumed by an application.
   no-specific-plan-parameters:
     tooltip: The service provider did not define specific parameters for the selected plan. Refer to the documentation to learn about the required parameters, and define them as a JSON in the editor.
+  statuses:
+    deprovisionedsuccessfully: Deprovisioned Successfully
+    deprovisionrequestunflight: Deprovision Request In Flight
```

In line 970 and 976, the variable is `deprovisionrequest**un**flight` but the label is `Deprovision Request In Flight`, shouldn't they both be `inflight`? 

```suggestion
    deprovisionrequestinflight: Deprovision Request In Flight
```

---

Damn!